### DISCUSSION TITLE
doom emacs, editorconfig, magit views seems to ignore correct tab width, how to fix?

### DISCUSSION THREAD

**Discussion by YellowOnion** [magit/magit#4848] [Support] [answered]
I have the proverbial `.editorconfig` in some of my projects, and the ones that have tab width of 8, diff view seems to use 4 width tabs, and worse still, in one of my projects inside the \*Messages\* buff there's messages explicitly stating it's setting my indentation to 4.

```.editorconfig
[*.{ktest,sh}]
indent_style = spaces
indent_size = 4
tab_width = 8
````
commit view (possible issue matching the file name `.ktest.~master~`)
![image](https://user-images.githubusercontent.com/364160/213035501-b48d36c2-1eae-4c11-ab61-82acead0c2af.png)
diff view:
![image](https://user-images.githubusercontent.com/364160/213035063-b4d0166f-8155-4fd8-b503-d481d80f9402.png)
reference:
![image](https://user-images.githubusercontent.com/364160/213035765-801f2317-dc54-489f-b338-4ac1f5c866b3.png)

**Comment by tarsius** [comment, marked as answer]
IMO it is editorconfig that should implement support for magit, not the other way around.
